### PR TITLE
Add quickstart for Front Door Standard/Premium

### PR DESCRIPTION
**Pull request by johndowns**

### CODE REVIEW

File: quickstart/101-front-door-standard-premium/providers.tf [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.27.0"
+    }
+
+    random = {
+      source = "hashicorp/random"
+    }
```
Comment (grayzu) [inline quickstart/101-front-door-standard-premium/providers.tf:12]: Please add version for random provider here.

File: quickstart/101-front-door-standard-premium/readme.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+| - | - |
+| `azurerm_resource_group` | The resource group for all the deployed resources.|
+| `azurerm_cdn_frontdoor_profile` | The Front Door profile. |
+| `azurerm_cdn_frontdoor_endpoint` | The Front Door endpoint. |
+| `azurerm_cdn_frontdoor_origin_group` | The Front Door origin group. |
+| `azurerm_cdn_frontdoor_origin` | The Front Door origin, which refers to the App Service app. |
+| `azurerm_cdn_frontdoor_route` | The Front Door route. |
+| `azurerm_service_plan` | The App Service plan. |
+| `azurerm_windows_web_app` | The App Service app. |
+
```
Comment (grayzu) [inline quickstart/101-front-door-standard-premium/readme.md:18]: Include the random resources here with description.

File: quickstart/101-front-door-standard-premium/readme.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+## Variables
+
```
Comment (grayzu) [inline quickstart/101-front-door-standard-premium/readme.md:20]: Would be good to document the defaults here as well.

File: quickstart/101-front-door-standard-premium/variables.tf [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+variable "front_door_sku_name" {
+  type    = string
+  default = "Standard_AzureFrontDoor"
+  validation {
+    condition     = contains(["Standard_AzureFrontDoor", "Premium_AzureFrontDoor"], var.front_door_sku_name)
+    error_message = "The SKU value must be Standard_AzureFrontDoor or Premium_AzureFrontDoor."
+  }
+}
+
+resource "random_id" "app_name" {
```
Comment (grayzu) [inline quickstart/101-front-door-standard-premium/variables.tf]: These two resources should not be in the variables.tf file. I suggest moving them to the resource-group.tf file.

**Review by grayzu — CHANGES_REQUESTED** [review]
Thanks for the PR!!

Looks great. I have requested a few minor changes and added one of our Terraform Front Door experts to do a quick review.

File: quickstart/101-front-door-standard-premium/providers.tf (hunk 1 above)
Comment (johndowns) [reply quickstart/101-front-door-standard-premium/providers.tf:12]: Updated.

File: quickstart/101-front-door-standard-premium/readme.md (hunk 2 above)
Comment (johndowns) [reply quickstart/101-front-door-standard-premium/readme.md:18]: Updated.

File: quickstart/101-front-door-standard-premium/variables.tf (hunk 4 above)
Comment (johndowns) [reply quickstart/101-front-door-standard-premium/variables.tf]: Updated.

File: quickstart/101-front-door-standard-premium/readme.md (hunk 3 above)
Comment (johndowns) [reply quickstart/101-front-door-standard-premium/readme.md:20]: Updated. Was this what you had in mind?

Comment (grayzu) [reply quickstart/101-front-door-standard-premium/readme.md:20]: Yup, that is perfect!

**Review by WodansSon — APPROVED** [review]
LGTM! 🚀

**Review by grayzu — APPROVED** [review]

**Pull request merged**